integ-test/src/test/resources/correctness/bugfixes/901.txt

```diff
@@ -0,0 +1,3 @@
+SELECT * FROM (SELECT Origin, Dest, AvgTicketPrice FROM kibana_sample_data_flights) AS flights
+SELECT * FROM (SELECT Origin AS origin FROM kibana_sample_data_flights) AS flights
+SELECT * FROM (SELECT * FROM (SELECT Origin FROM kibana_sample_data_flights) AS flights) AS f
```

Comment:
Could you add one more test case for `SELECT fields FROM (SELECT * FROM ...) ?

Reply:
Sure, done. Thanks!